apps/main-cf.yml.j2

```diff
@@ -245,7 +245,7 @@ Resources:
             AllowedOrigins:
               - "*.asf.alaska.edu"
 
-  {% if security_environment != 'EDC' %}
+  {% if security_environment in ('ASF', 'JPL-public') %}
```

Comment:
I was able to deploy into the new `hyp3-a19-jpl` environment by removing this bucket policy